docs/users/subscriptions/index.md

```diff
@@ -12,3 +12,52 @@ links:
         - /docs/users/subscriptions/cancel-subscription
     featured:
 ---
+
+## About billing for Devopness plans and actions
+Devopness usage is free for everyone, without any limits on the amount of projects, environments or the amount of resources that can be added to each infrastructure environment.
+
+Paid subscription plans are available for users and teams who want to increase their productivity by taking the most of Devopness additional features.
+
+Independently of the subscription plan of a Devopness project, the main usage is controlled by credits.
+> Devopness consumes one credit per action step completed in a project pipeline
+
+Each Devopness user account receives a certain amount of credits monthly, depending on the user’s active subscription plan.
+
+Credits are consumed from the project owner’s account: if you are collaborating on somebody else’s project as a team member, but you’re not the owner of that project, then your credits will not be consumed when working on that project.
+
+For details, see Usage limits and our {% mentionPost "/docs/users/subscriptions/faq" %}
+
+### Billing date and subscription renewal
+You can view your account's subscription details, payment history and your next billing date in your account's billing settings at {% mentionPost "/docs/users/subscriptions/upgrade-subscription" %}
+
+## Billing settings
+You can manage your billing settings, such as subscription plans and payment methods in your account's billing settings at {% mentionPost "/docs/users/subscriptions/upgrade-subscription" %}
+
+## Overages
+At the moment Devopness does not allow usage-based billing, so user projects cannot consume more credits than the amount included in the user’s active subscription plan.
+
+There will be no surprises in your monthly or annual billing, as Devopness will charge you upfront for the included amounts of credits in your subscription plan.
+
+You can see the amount of credits included in your account's billing settings at Billing & Plans 
+
+## Usage limits
+There are some limits when using Devopness actions, depending on the subscription plan.
+These limits are subject to change every time our plans are updated. For details, see our pricing page https://www.devopness.com/pricing/
```

Comment:
The domain can be removed using markdown links

```suggestion
These limits are subject to change every time our plans are updated. For details, see our [pricing page](/pricing)
```

this pattern is also applied here
 https://github.com/devopness/devopness/blob/bc0246e755845541a04b269d73e2f225e03dd0f3/docs/users/subscriptions/faq/index.md?plain=1#L22